gpt4all-bindings/rust/src/wrappers/loader/domain.rs

```diff
@@ -0,0 +1,159 @@
+use std::fmt;
+
+#[derive(Debug, Clone)]
+pub struct CompletionModelConfig {
+    pub default_prompt_template: Option<String>
+}
+
+#[derive(Debug, Clone)]
+pub enum ModelBuildVariant {
+    /// Automatically select the appropriate build variant.
+    AUTO,
```

Comment:
Clippy is likely to complain about naming here. Would you be okay with making these more Rust-like (`Auto`, `Default`, `AvxOnly`, ...)?